docs/design/proposals/rbd-pv-key-rotation.md

```diff
@@ -0,0 +1,77 @@
+# Encryption Key Rotation
+
+## Proposal
+
+Subject of this proposal is to add support for rotation of
+encryption keys (KEKs) for encrypted volumes in Ceph-CSI.
+
+Support for rotating keys on RWX/ROX volumes is out of scope for now
+and shall be added later.
```

Comment:
Also add that filesystem encryption with `fscrypt` is out of scope, this is only for block-device encryption with LUKS. The RBD backend supports both, CephFS only supports `fscrypt`, NFS supports none.